ipfs-cluster/trigger-server/README.md

```diff
@@ -1,3 +1,9 @@
+# What is trigger server
+
+The trigger server's purpose is defined in ipfs/ipfs-cluster/issues/244. The goal is to create an endpoint
+which can be triggered remotely and automated to run the tests, obtaining their results.
+
+
 To build RPM/DEB
 ---
 
```

Comment:
As an additional note, we don't need new issues for every commit addressing issues in a PR. Commit messages should be more descriptive than this, but it's fine too if you just write `WIP`, and at the end of the review process squash it all together in a single commit, but in any case an additional issue is not needed.